dotnet/src/Backends.HuggingFace/HttpSchema/CompletionRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Backends.HuggingFace.HttpSchema;
+
+/// <summary>
+/// HTTP schema to perform completion request.
+/// </summary>
+[Serializable]
+public sealed class CompletionRequest
+{
+    /// <summary>
+    /// Prompt to complete.
+    /// </summary>
+    [JsonPropertyName("prompt")]
+    public string? Prompt { get; set; }
```

Comment:
Is there any situation where we need to send no prompt and/or no model name?

I would use an empty string by default, rather than nullable, same for the field below.